dashboard/db/schema.rb

```diff
@@ -191,6 +191,20 @@
 
   add_index "games", ["intro_video_id"], name: "index_games_on_intro_video_id", using: :btree
 
+  create_table "hint_view_requests", force: :cascade do |t|
```

Comment:
The table indexing isn't exactly what I had in mind. My thought would be to index on the pair (script_id, level_id) rather than on them individually. Perhaps even index on the triple (script_id, level_id, user_id) rather than the pair (script_id, level_id). Alternately, index on the triple (user_id, script_id, level_id), depending on whether we think we'll be aggregating by user (e.g., studying how many hints given users use) versus aggregating by script-level (e.g., studying how many hints given levels require).